frontend/src/app/modules/result-selection/components/measurands/measurand-select/measurand-select.component.html

```diff
@@ -1,5 +1,12 @@
 <select class="form-control" [(ngModel)]="selectedMeasurand" (ngModelChange)="selectMeasurand()"
         [compareWith]="compareMeasurands">
+  <optgroup label="{{'frontend.de.iteratec.osm.performance-aspects' | translate}}">
+    <ng-container *ngFor="let type of perfAspectTypes$ |async">
+      <option>
```

Comment:
```suggestion
      <option *ngFor="let type of perfAspectTypes$ | async">
```
You should do the ngFor 'for' each option that you create. ng-container is redundant because it is not loaded in the DOM and just creates the option element, which you could create directly.